examples/dataproc-spanner/jdbc/README.md

```diff
@@ -0,0 +1,198 @@
+## BigQuery to Spanner using Apache Spark in Scala
+
+### Why create this repo ? 
+
+ * Google Cloud customers need to move data from BigQuery to Spanner using Apache Spark in Scala
```

Comment:
If my understanding is correct, the actual JDBC write code only requires a Dataframe. Instead of limitating ourselves to just BigQuery to Spanner, which could sound strange given that data usually flow the other way around, can we say that this is provide generic Spanner JDBC write capability in Spark Scala. We can say that we are demonstrating the capability using a Dataframe generated from reading a BigQuery table as an example.